sbol/test/test_identified.py

```diff
@@ -150,5 +150,176 @@ def test_was_generated_by2(self):
         self.assertEqual(cd.wasGeneratedBy, expected)
 
 
+class TestCopy(unittest.TestCase):
```

Comment:
this test class doesn't appear to be tied in to the test functions in `sbol/test/__init__.py`. Should it be? I think so, unless it takes a very long time to run.

Reply:
Yes, inserted it into the list of tests in 11fce9e